homeassistant/components/media_player/plex.py

```diff
@@ -114,10 +119,15 @@ def setup_plexserver(host, token, hass, add_devices_callback):
     plex_clients = {}
     plex_sessions = {}
 
-    @util.Throttle(MIN_TIME_BETWEEN_SCANS, MIN_TIME_BETWEEN_FORCED_SCANS)
```

Comment:
Why did you remove the Throttle? This means that if you have 10 clients that every 15 seconds all of them will hit the server for the same data.